docs/volunteers/understand-students.md

```diff
@@ -0,0 +1,55 @@
+
+Some of our previous asylum seeker students have included:
+Max (not a real name) went from asylum seeker to refugee. Max had a tech background, but suffered from social anxiety and couldn't find a job. CYF helped them gain confidence and the tech skills needed to work in the local tech industry. Once their refugee status was granted, they found a good job in tech.
+
+### Do's and don'ts for interacting with these groups
+DO
+- Treat each student as a complete and complex human. They are more than their trauma.
+- Treat each student as you would any other student you might teach.
+- Behave in a manner that shows and proves you are approachable, trustworthy and safe.
+- Be patient. It's important to understand that people's situations may be very difficult. Some people do have to contend with internet issues, for example, or have more severe issues with health or finances linked to their current situation.
+
+DON'T
+- Ask invasive questions about their past experiences or reasons for leaving their country of origin. It is none of your business and they do not owe you their story. Think. Would you ask someone you barely know about a trauamtic event like sexual abuse or a bereavement before you had even had a conversation with them?
+- Betray the trust of students. When you say you will do something, do it. This can be as small as getting back to them about homework or solving a big issue for them. Building trust in the community is extremely important.
+- Assume you know everything about being an asylum seeker or refugee. Each case is different and no amount of research will replace a real, lived experience.
+
+### Resources to learn more or support refugees and asylum seekers
+There are lots of incredible foundations and groups out there making information available for asylum seekers and refugees, as well as supporting them in a myriad of ways. Some of the groups even work with CodeYourFuture to help us find the students who want to study with us. Here are some places you can start your research:
+[Right To Remain](https://righttoremain.org.uk/)
+[City of Sanctuary](https://cityofsanctuary.org/)
+[Refugee Action](https://www.refugee-action.org.uk/)
+[Refugee Council](https://www.refugeecouncil.org.uk/)
+[UNHCR UK](https://www.unhcr.org/uk/asylum-in-the-uk.html)
+
+There are many local groups with whom we work alongside or who you can get involved with to help your community. We will continue to add to this section to make these known.
```

Comment:
Whom is usually the object of the sentence, so I would say this is correct usage, but you're right that it can come across stiff due to it being a bit 'old-fashioned'. I've changed the structure of the sentence.